qiskit/quantum_info/operators/channel/kraus.py

```diff
@@ -419,7 +419,7 @@ def _tensor_product(self, other, reverse=False):
             if kb_r is None:
                 kb_r = kb_l
             if reverse:
-                kab_r = [np.kron(b, a) for a in ka_r for b in kb_r]
+                kab_r = [np.kron(a, b) for b in ka_r for a in kb_r]
```

Comment:
Same as above

Reply:
Ok, updated in: https://github.com/Qiskit/qiskit-terra/pull/3635/commits/689c413d276bb5c1934b4537964d8cd331a0bd54 to use the original ordering but changed the variable names to be `a_in` and `b_in` so pylint doesn't complain